### ISSUE TITLE
Bumping from `v2.0.2` to `v2.1.0` raises `Error: Cannot find module './impl/format'`

### ISSUE THREAD

**Issue by simensol** [jakebailey/pyright-action#88]
After bumping from `v2.0.2` to `v2.1.0` I now get:

```bash
Run jakebailey/pyright-action@v2.1.0
  with:
    working-directory: backend
    version: 1.1.348
    no-comments: false
    warnings: false
    lib: false
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.11.8/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.8/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.8/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.8/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.8/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.8/x64/lib
node:internal/modules/cjs/loader:1051
  throw err;
  ^
Error: Cannot find module './impl/format'
Require stack:
- /home/runner/work/_actions/jakebailey/pyright-action/v2.1.0/dist/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at /home/runner/work/_actions/jakebailey/pyright-action/v2.1.0/dist/index.js:4282:25
    at /home/runner/work/_actions/jakebailey/pyright-action/v2.1.0/dist/index.js:4272:17
    at node_modules/jsonc-parser/lib/umd/main.js (/home/runner/work/_actions/jakebailey/pyright-action/v2.1.0/dist/index.js:4278:7)
    at __require (/home/runner/work/_actions/jakebailey/pyright-action/v2.1.0/dist/index.js:12:50)
    at Object.<anonymous> (/home/runner/work/_actions/jakebailey/pyright-action/v2.1.0/dist/index.js:8109:35)
    at Module._compile (node:internal/modules/cjs/loader:1241:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/jakebailey/pyright-action/v2.1.0/dist/index.js'
  ]
}
Node.js v20.8.1
```

when running

```yml
- name: Run Pyright
  uses: jakebailey/pyright-action@v2.1.0
  with:
      working-directory: "backend"
      version: 1.1.348
```

Downgrading to `v2.0.2` fixes the issue:

```bash
Run jakebailey/pyright-action@v2.0.2
  with:
    working-directory: backend
    version: 1.1.348
    no-comments: false
    warnings: false
    lib: false
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.11.8/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.8/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.8/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.8/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.8/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.8/x64/lib
/usr/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/work/_temp/c4b17f4f-5b08-4e65-8fd7-59db68d2ec54 -f /home/runner/work/_temp/e2383b4c-ca59-4abf-b8c8-ab0fba725d0e
pyright 1.1.348, node v20.8.1, pyright-action 2.0.2
Working directory: /home/runner/work/project/project/backend
Running: /home/runner/runners/2.313.0/externals/node20/bin/node /opt/hostedtoolcache/pyright/1.1.348/x64/package/index.js --outputjson
0 errors, 0 warnings, 0 informations
```

**Comment by jakebailey** [comment]
Wow, very sorry about this, will try and fix it.

**Comment by jakebailey** [comment]
Fixed in v2.1.1. Super sorry; I should have had some sort of smoke test to verify that things work outside of tests. Now I do!
